File: PA - PROGRAMAÇÃO E ALGORITIMOS/python - Valentino e Caua/python-conversion/cpp/ex10_sistema_senha.cpp
/*
   Autores: Caua Balzaneli, Valentino Hoehne
   Data: 06/08/2026
   Finalidade: Simula um sistema simples de senha com até 3 tentativas.
*/

#include <iostream>
using namespace std;

int main()
{
    int senha;
    int senhaCorreta = 1234;
    int tentativas = 1;
    int limiteTentativas = 3;

    cout << "Digite a senha: ";
    cin >> senha;

    while (senha != senhaCorreta && tentativas < limiteTentativas)
    {
        cout << "Senha incorreta. Tente novamente: ";
        cin >> senha;

        tentativas++;
    }

    if (senha == senhaCorreta)
    {
        cout << "Acesso liberado." << endl;
    }
    else
    {
        cout << "Acesso bloqueado." << endl;
    }

    return 0;
}

File: PA - PROGRAMAÇÃO E ALGORITIMOS/python - Valentino e Caua/python-conversion/python/ex10_sistema_senha.py
"""
Autores: Caua Balzaneli, Valentino Hoehne
Data: 06/08/2026
Finalidade: Simula um sistema simples de senha com até 3 tentativas.
"""

senha = int(input("Digite a senha: "))
senha_correta = 1234
tentativas = 1
limite_tentativas = 3

while (senha != senha_correta and tentativas < limite_tentativas):
    print("Senha incorreta. Tente novamente: ")
    senha = int(input("Digite a senha: "))

    tentativas = tentativas + 1

if (senha == senha_correta):
    print("Acesso liberado.")

else:
    print("Acesso bloqueado.")
